src/pages/[id].page.jsx

```diff
@@ -23,6 +24,26 @@ export default function Details({ details }) {
     alt: `${componentName} image`,
   };
 
+  const [htmlCode, setHtmlCode] = useState('');
+  const [cssCode, setCssCode] = useState('');
+  const [javascriptCode, setJavascriptCode] = useState('');
+
+  // const htmlCode = import(`../code-examples/${slug}/html.js`);
+  // dynamic imports need to be asynchronous
+  useEffect(() => {
+    const updateCodeBlocks = async () => {
+      const html = await import(`../code-examples/${slug}/html.js`);
+      setHtmlCode(html.default);
+
+      const css = await import(`../code-examples/${slug}/css.js`);
+      setCssCode(css.default);
+
+      const javascript = await import(`../code-examples/${slug}/javascript.js`);
+      setJavascriptCode(javascript.default);
```

Comment:
Out of curiosity: what is the `.default` doing? I see you have them on for each HTML, CSS & JS, but I don't know what that does.

Reply:
So what is happening here is when you go in and inspect what gets pulled in there w/ browser inspector tool you can see that the name is default. I threw in a console log in the useEffect so you can see it. If you look in the browser snipped, you can see that "default" is the name of the value you want to when you set the new code snippet:
![image](https://user-images.githubusercontent.com/49820744/200865981-d754418d-6afa-4822-9d86-c06eaa2c1032.png)


![image](https://user-images.githubusercontent.com/49820744/200865880-8ea2fadd-016a-4e49-b219-6dd28c13336a.png)

Reply:
I'm referring to the default: "tabs HTML" 
![image](https://user-images.githubusercontent.com/49820744/200866182-d4b0b962-210e-4049-8642-7e2ead14c67f.png)

Reply:
Huh, cool! I'll have to keep that in mind as something to look out for. Thanks!